core-web/src/main/java/org/silverpeas/core/webapi/variables/VariableEntity.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (C) 2000 - 2018 Silverpeas
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * As a special exception to the terms and conditions of version 3.0 of
+ * the GPL, you may redistribute this Program in connection with Free/Libre
+ * Open Source Software ("FLOSS") applications as described in Silverpeas's
+ * FLOSS exception.  You should have received a copy of the text describing
+ * the FLOSS exception, and it is also available here:
+ * "https://www.silverpeas.org/legal/floss_exception.html"
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.silverpeas.core.webapi.variables;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.silverpeas.core.variables.Variable;
+import org.silverpeas.core.variables.VariableScheduledValue;
+import org.silverpeas.core.webapi.base.WebEntity;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlRootElement;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.PROPERTY)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class VariableEntity implements WebEntity {
+
+  private URI uri;
+
+  private String id;
+  private String label;
+  private String description;
+  private String value;
+  private List<VariableScheduledValueEntity> values = new ArrayList<>();
+
+  protected VariableEntity() {
+  }
+
+  public static VariableEntity fromVariable(Variable variable) {
+    VariableEntity entity = new VariableEntity().decorate(variable);
+    List<VariableScheduledValueEntity> periods = variable.getVariableValues()
+        .stream()
+        .map(v -> VariableScheduledValueEntity.fromVariableScheduledValue(v))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1612)